src/plugins/intel_gna/tests/unit/ops/util_test.cpp

```diff
@@ -0,0 +1,156 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <cstdint>
+#include <vector>
+#include <thread>
+#include <memory>
+
+#include <gtest/gtest.h>
+#include "ops/util/util.hpp"
+
+using namespace ov::intel_gna::ngraph_util;
+
+namespace test {
+
+using GnaOpsUtilTestParams = std::tuple<std::shared_ptr<ngraph::Node>, bool>;
+
```

Comment:
Here and later consider using ov instead of ngraph namespace. If you want you can add
using namespace ov;
using namespace ov::opset9;

Reply:
namespace ov::opset9 is absent